### DISCUSSION TITLE
grid控件下hscrollbar和vscrollbar的showbutton2重叠

### DISCUSSION THREAD

**Discussion by WXX0309** [netease-im/NIM_Duilib_Framework#430] [Q&A]
现在就是横的滚动条的button2 被竖的盖在上面了。我看了其他控件也没有这个问题，有什么好的解决方案吗？目前需求是可以不需要竖的button1，但如果不设置的话，滚动条本身也会遮住另一个的button2

**Comment by nmgwddj** [comment]
grid 是社区开发者贡献，您可以参考源码尝试修改。
